README.md

```diff
@@ -51,36 +61,55 @@ This will start training on a GPU, if available. If you want to use a CPU instea
 
 Replace the models.py with your own if you want to use this code as a scaffold. Feel free to use only the data loaders and metrics and write your own main loop etc., but we may be slower to evaluate your PR the more different it is from this code.
 
-## Task Details
+### Task Details
 
-For a full write-up of this task, see [this document](TODO).
+This is framed as a supervised learning task, with the following inputs, targets, and metrics:
 
-**Input:** TODO
+Inputs and targets are described for 1 data example (1 person).
 
-**Targets:** TODO
+**Input:** 
+* `reported_symptoms`: (*14, N_s*) array where *N_s* is the number of possible symptoms. Each (*i,j*) element of the array is a binary indicator of whether the person had symptom *j* at day *t-i*
+* `test_results` (14) array of values {-1,0,1} where the *i*th element indicates the test result at day *t-i*. A value of -1 means tested negative, 0 means not tested, and 1 means tested positive.
+* `candidate_encounters`: (*N_e, 3*) array where *N_e* is the number of encounters in the past 14 days. For each encounter, the 3 dimensions are: 
+    - `ID`: an integer indicating the identity of the other person in the encounter (estimated by a [clustering algorithm](TODO))
```

Comment:
We can add a link to the clustering algorithm document here: https://www.overleaf.com/read/skcfppcjmccm
It's WIP, but what isn't